NEWS.md

```diff
@@ -8,13 +8,18 @@
 
 - Added new boostrap functionality for models with constant or variable returns to scale based on Simar and Wilson (1998) (#6)
 - Added new function `unset_env_vars()` that will clear environment variables set by pioneeR if the app exits unexpectedly and the variables are not unset automatically
+- Added new function `compute_scale_efficiency` to run CRS, VRS and NIRS models to compute scale efficiency and evaluate optimal scale size for each unit
+- Added new function `summary_tbl_dea` to create a summary table of efficiency scores from a vector of efficiency scores or an object of type Farrell as returned from the `Benchmarking::dea` function
+- Added new helper function `create_matrix` that will create a matrix of input values or output values based on a data.frame
```

Comment:
Very small comment, but I think the convention for R functions is to include the `()` at the end of each function name, eg. summary_tbl_dea**()**

Reply:
Fixed now